### PR TITLE
Add federated search and change header style

### PR DESCRIPTION
**Pull request by channprj**
## Summary

- Add federated search based on existing transaction and block pages (https://github.com/planetarium/libplanet-explorer-frontend/issues/5#issuecomment-546578989)
- Change header style (#36)
  - Refer to [docs.libplanet.io](https://docs.libplanet.io)
  - Add margin-bottom
- Show error message
- Add default eslint config

## Screenshots

<details>
  <summary>Add federated search and error message</summary>
  Screenshot 2019-10-30 01 51 07
</details>

<details>
  <summary>Change header style</summary>
  Screenshot 2019-10-30 01 39 40
  Screenshot 2019-10-30 01 50 22
</details>

## What to do next

- Spacing adjustment
- Add footer
- Dropdown search filter (like a etherscan)
- Turn `alert()` into toast message (or modal)
- Error exceptions
- Add favicons, manifest.json, etc.

## Reference

- https://etherscan.io
- https://bloks.io
- https://eospark.com
- https://www.mintscan.io

### CODE REVIEW

**Comment by limebell** [comment]
It seems like search bar isn't behave as you described―instead of call `/search/?{value}`, it just calls `/block/?{value}`. Is there any switch to activate?

**Comment by moreal** [comment]
@channprj the commit including favicons seems missed. isn't it?

**Comment by channprj** [comment]
@limebell It seems that one commit is missing with `Layout.tsx`. Maybe I was so tired at that time... 😂I'll fix it after work. Thank you.

@moreal Favicon is not included in this pr. It's just what to do next. Sorry for the confusion.

**Comment by channprj** [comment]
I recover stashed uncommitted changes. 😅

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
   DirectionalHint,
 } from 'office-ui-fabric-react/lib/Tooltip';
 import { getId } from 'office-ui-fabric-react/lib/Utilities';
-import Logo from '../static/img/logo.svg';
+const logo = require('../static/img/logo.svg');
```
Comment (channprj) [inline src/components/Layout.tsx]: There is another option like below:

```ts
declare module "*.svg" {
  const content: any;
  export default content;
}
```

But I think writing custom definition is not necessary for this case. Anyway, I will follow your rules. 🤗

## Reference

- [Webpack - Importing Other Assets](https://webpack.js.org/guides/typescript/#importing-other-assets)

**Pull request merged**